tests/inspectdb/tests.py

```diff
@@ -48,7 +48,10 @@ def assertFieldType(name, definition):
         return assertFieldType
 
     def test_field_types(self):
-        """Test introspection of various Django field types"""
+        """
+        Test introspection of various Django field types.
+        See also postgres_tests.test_introspection for PostgreSQL-specific fields.
```

Comment:
I don't think this change is needed. For example, there isn't a similar comment in the model field and form field tests.